### ISSUE TITLE
`piece()` no longer works with `{rayrender}` 0.34.2

### ISSUE THREAD

**Issue by trevorld** [piecepackr/piecepackr#344]
> Dear maintainer,
> 
> Please see the problems shown on
> <https://cran.r-project.org/web/checks/check_results_piecepackr.html>.
> 
> Please correct before 2024-08-19 to safely retain your package on CRAN.
> 
> It seems we need to remind you of the CRAN policy:
> 
> 'Packages which use Internet resources should fail gracefully with an informative message
> if the resource is not available or has changed (and not give a check warning nor error).'
> 
> This needs correction whether or not the resource recovers.
> 
> The CRAN Team

* Note CRAN submissions will be down Aug 1, 2024 to Aug 16, 2024 due to CRAN vacation.
* Relevant "donttest" "Additional Issues" check is here: https://www.stats.ox.ac.uk/pub/bdr/donttest/piecepackr.out
* Although BDR suggests this may be due to an Internet resource issue however {piecepackr} doesn't use any Internet resources.
* This actually seems to be due to a regression introduced by the latest version of the suggested package {rayrender} which is used in an example wrapped by `\donttest{}`

**Comment by trevorld** [comment]
```r
library("piecepackr")
cfg <- game_systems("sans3d")$piecepack
rayrender::render_scene(piece("tile_face", suit = 3, rank = 3, cfg = cfg))
```
```r
Error: Loading of '/tmp/rtmpzgulep/working_dir/rtmpmja75i/file204e035b63d10.png' failed due to error: can't fopen-- nx/ny/channels :0/0/0
```

**Comment by trevorld** [comment]
Opened an upstream issue with minimal, reproducible example: https://github.com/tylermorganwall/rayrender/issues/51

**Comment by trevorld** [comment]
* Bug fixed upstream in `{rayrender}` v0.34.3 which was successfully published on CRAN 
* We still need to bump the version number of `{rayrender}` to v0.34.3 in our `DESCRIPTION` file (and then submit to CRAN before deadline after submissions are re-allowed)
